client/src/components/Column.js

```diff
@@ -4,9 +4,33 @@ import Project from './Project';
 import PropTypes from 'prop-types';
 
 class Column extends Component {
+  constructor(props) {
+    super(props);
+    this.dragOver = this.dragOver.bind(this);
```

Comment:
Oh, I thought React handled this for you when you assigned them as attributes… 🤔

Reply:
I'm not sure what you mean by this

Reply:
From https://reactjs.org/docs/faq-functions.html#why-is-binding-necessary-at-all:
> With React, typically you only need to bind the methods you pass to other components.
It looks like you're calling `this.dragOver`, not passing it, so shouldn't need to bind it…

Reply:
So without the .bind() it will get called, but in the wrong context - resulting in the error `TypeError: this is undefined; can't access its "props" property`

Reply:
Thank you for the link! it shows `<button onClick={() => this.handleClick(id)} />` as an example of passing params to a function, which helps so much! I was trying to avoid using bind in the html elements, and so was creating some almost-empty functions.